README.md

```diff
@@ -1,2 +1,50 @@
-# static-code-analysis-tool
-Tool for performing static code analysis for Ballerina projects.
+# Ballerina Static Code Analysis Tool
+
+## Overview
+Static Code Analysis (SCA) is an approach that uses tools to examine code without execution. 
+It's used for identifying potential issues like bugs, vulnerabilities and style violations.
+SCA improves software quality by early issue detection, creating better maintainability, and
+providing enhanced security. The Ballerina scan tool provides a set of command line tools to
+statically analyze Ballerina files and report analysis issues.
+
+## Prerequisites
+1. openJDK 17 ([Adopt OpenJDK](https://adoptium.net/temurin/releases/?version=17) or any other OpenJDK distribution)
+
+2. [Ballerina version: 2201.8.5](https://ballerina.io/downloads/)
+
+
+## Building from the source
+Execute the commands below to build from the source.
+
+1. Export GitHub Personal access token with read package permissions as follows,
+    ```bash
+    export packageUser=<Username>
+    export packagePAT=<Personal access token>
+    ```

```

Comment:
Shall we have the steps for windows OS as well? It'll make things easier for the windows users.

Reply:
+1, Fixed in [cf88522](https://github.com/ballerina-platform/static-code-analysis-tool/pull/2/commits/cf88522631113897d629d7a51feece25f60b9303)